packages/react/src/ImageSlider/ImageSlider.tsx

```diff
@@ -126,30 +188,23 @@ export const ImageSliderRoot = forwardRef(
       setAtEnd(last === sliderScroller?.children[currentSlideId])
     }
 
-    const handleKeyDown: KeyboardEventHandler<HTMLDivElement> = (e) => {
-      if (e.key === 'ArrowRight') {
-        goToNextSlide()
-      } else if (e.key === 'ArrowLeft') {
-        goToPreviousSlide()
-      }
-    }
-
     return (
       <ImageSliderContext.Provider value={{ currentSlide: currentSlideId, goToSlideId }}>
         <div
           {...restProps}
           ref={ref}
+          aria-roledescription="carousel"
+          tabIndex={-1}
           className={clsx(
             'ams-image-slider',
             controls && 'ams-image-slider--controls',
             scrollbar && 'ams-image-slider--scrollbar',
-            snapstop && 'ams-image-slider--snapstop',
             thumbnails && 'ams-image-slider--thumbnails',
             className,
           )}
         >
           {controls && (
-            <div className="ams-image-slider__controls" onKeyDown={handleKeyDown}>
+            <div className="ams-image-slider__controls">
```

Comment:
We could extract a small component for these buttons as well, just like for the scroller and the thumbnails.

Reply:
An ImageSliderControlsButton you mean or the whole controls component?